src/views/projects/Patch.svelte

```diff
@@ -156,7 +156,7 @@
         type: "review",
         inner: [rev.id, review.author.id, review],
       })),
-      ...rev.merges.map<TimelineMerge>(inner => ({
+      ...patch.merges.map<TimelineMerge>(inner => ({
```

Comment:
The second index of the `timelineTuple` variable is a aggregated array of revisions.
So this works as intended we will need to filter the patch merges by the revisionId, so this would end up being something like this
```ts
...patch.merges.filter(merge => merge.revision === rev.id).map<TimelineMerge>(inner => ({
```

Reply:
[fixed](https://github.com/radicle-dev/radicle-interface/compare/7152476b8803f8d00304d6c5e0ce9b3fed13da84..b1fbaf3818b3462ead0c80a22b6e8266646330f3)